root/report/components/CdTocList.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 Jerome Roy
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import PaginatedResults from '../../components/PaginatedResults';
+import loopParity from '../../utility/loopParity';
+import type {ReportCdTocT} from '../types';
+import formatTrackLength
+  from '../../static/scripts/common/utility/formatTrackLength';
+
+type Props = {
+  +items: $ReadOnlyArray<ReportCdTocT>,
+  +pager: PagerT,
+};
+
+const CdTocList = ({
+  items,
+  pager,
+}: Props): React.Element<typeof PaginatedResults> => {
+  const colSpan = 3;
+
+  return (
+    <PaginatedResults pager={pager}>
+      <table className="tbl">
+        <thead>
+          <tr>
+            <th>{l('Disc ID')}</th>
+            <th>{l('Format')}</th>
+            <th>{l('Length')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {items.map((item, index) => {
+            return (
+              <tr className={loopParity(index)} key={item.cdtoc_id}>
+                {item.discid ? (
+                  <>
+                    <td>
+                      <a href={'/cdtoc/' + item.discid}>
```

Comment:
This could be integrated into EntityLink instead

Reply:
See CDTocLink

Reply:
Uh, not sure why I didn't see it. Fixing now

Reply:
BTW https://github.com/metabrainz/musicbrainz-server/blob/master/root/types.js#L753 mentions an undefined MediumCDToc that you created last month, you want me to fix it at the same time?

Reply:
In a separate commit, but sure! (if it doesn't break anything else)

Reply:
Just pushed, let's see if tests complain